src/index.html

```diff
@@ -13,6 +13,48 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image"></div>
+
+      <div class="card__info">
+        <h2 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <h3 class="card__code">Product code: 195434</h3>
+
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+
+            <div class="stars__star"></div>
+          </div>
+
+          <div>
+            <span class="card__review">Reviews: 5</span>
+          </div>
+        </div>
+
+        <div class="card__prices">
+          <span class="card__price text">Price:</span>
```

Comment:
The class name 'text' is too generic. Consider a more specific class name that describes its purpose or follows BEM conventions.